.github/demo-dashboard/Dockerfile

```diff
@@ -30,12 +35,4 @@ RUN curl -fsSL https://get.pnpm.io/install.sh | bash -
 ENV PNPM_HOME /root/.local/share/pnpm
 ENV PATH "$PNPM_HOME:$PATH"
 
-# nodejs seems to have changed the official mirror, the default in pnpm is very slow now
-RUN pnpm config -g set 'node-mirror:release' https://nodejs.org/dist && pnpm env use --global lts
-
-RUN npm install --global serve
-ENV PATH "$(npm bin -g):$PATH"
-
-COPY inject-env.sh /opt/inject-env.sh
-
-ENTRYPOINT ["bash", "-c", ". /opt/inject-env.sh && npx serve -s /opt/dashboard"]
+ENTRYPOINT ["pnpx", "serve", "-s", "/opt/dashboard"]
```

Comment:
`node` is required, and then `serve` is required to be installed too

Reply:
Thanks for helping to test! Should be fixed with https://github.com/open-rmf/rmf-web/pull/1007/commits/e05547f9cc3d85cbdb7e500aa0ba0a0f7566076a.